pkg/utils/client/client.go

```diff
@@ -20,12 +20,11 @@ import (
 	"errors"
 
 	openapi "github.com/alibabacloud-go/darabonba-openapi/v2/client"
-	ecsclient "github.com/alibabacloud-go/ecs-20140526/v4/client"
 	"github.com/alibabacloud-go/tea/tea"
 	aliyunconfig "github.com/aliyun/aliyun-cli/config"
 )
 
-func NewECSClient() (*ecsclient.Client, error) {
+func NewClientConfig() (*openapi.Config, error) {
```

Comment:
How about creating `NewECSClient`, `NewVPCClient`, `NewClientConfig`?

Reply:
`NewECSClient = ecs.NewClient(clientConfig)` Just one line. I don't think there is any need to wrap the function.

Reply:
ok

/cc @jwcesign for final check

Reply:
it's ok to keep like this.